### PR TITLE
Previous and next page buttons now have have responsive design in collections

### PR DESCRIPTION
**Pull request by bouhormq**
### Problem
https://tickets.metabrainz.org/browse/BB-579

### Solution
By changing the order in which the buttons are rendered, whenever the screen gets smaller, now the next and previous buttons are aligned and also added some margin in the bottom so that way there is some space between the lower level buttons and the top ones.

Before
BB
 After
BBetter (1)

### Areas of Impact
Only ascetic ones in the collections component.

### CODE REVIEW

File: src/client/stylesheets/style.less [hunk 1]
```diff
@@ -414,6 +414,10 @@ hr.wide {
 	}
 }
 
+.next, .previous > a {
```
Comment (MonkeyDo) [inline src/client/stylesheets/style.less:417]: This applies differently for next and for previous: you target elements with the .next class as well as <a> elements inside a parent with the class .previous (but not the element with the .previous class).

Using a percentage here can be tricky in some situations; instead let's use `1em` (or 0.7em, or whatever number you feel is right, but as an `em` value).

I would also like to make sure this applies only where we want it to, so altogether I suggest something like this: 
```suggestion
#pageWithPagination {
  .next, .previous {
    > a {
      margin-bottom: 1em;
    }
  }
}
```

File: src/client/entity-editor/common/sort-name-field.tsx [hunk 2]
```diff
@@ -166,7 +166,7 @@ function SortNameField({
 			label={label}
 			ref={(node) => { input = node; }}
 			tooltipText="Alphabetical sorting name. Examples: 'Dickens, Charles', 'Christmas Carol, A'.
-			<br>You can try to fill it automatically with the guess button"
+			You can try to fill it automatically with the guess button"
```
Comment (MonkeyDo) [inline src/client/entity-editor/common/sort-name-field.tsx:169]: This should not be part of this PR, you'll need to revert this change (and open another PR if you want to fix it ;) )

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
Thanks for opening the PR !

It's a really nice and simple trick to solve that problem !
How did you find that ? It was under our noses the whole time :D

File: src/client/entity-editor/common/sort-name-field.tsx (hunk 2 above)
Comment (bouhormq) [reply src/client/entity-editor/common/sort-name-field.tsx:169]: Sure thing!

File: src/client/stylesheets/style.less (hunk 1 above)
Comment (bouhormq) [reply src/client/stylesheets/style.less:417]: Perfect! I'll do some testing to see if it only applies to those exact buttons

**Comment by MonkeyDo** [comment]
@bouhormq Hi! Let me know if you have some time to finish this PR, otherwise do let me know and I'll finish it :)

**Comment by MonkeyDo** [comment]
Closing this as it is getting stale; please feel free to comment if you want to finish it and I'll reopen the PR :)
